gamemodes/terrortown/gamemode/client/cl_targetid.lua

```diff
@@ -308,6 +308,11 @@ function GM:HUDDrawTargetID()
         return
     end
 
+    -- if the entity also is a focused marker vision element, then targetID should be hidden
+    if ent == markerVision.GetFocussedEntity() then
```

Comment:
```suggestion
    if ent == markerVision.GetFocusedEntity() then
```

Seems like this slipped through, optional if you want i'd appreciate if we can fix this one too